source/sync/data-model/update-data-model/update-schema-development-apps.txt

```diff
@@ -0,0 +1,191 @@
+.. _dev-mode-breaking-changes:
+
+=======================================================
+Automatically Update a Data Model with Development Mode
+=======================================================
+
+.. meta::
+   :description: Update your Device Sync data model from your client app using Development Mode.
+
+.. facet::
+  :name: genre
+  :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+While your app is in development, you can leverage Development Mode to simplify
+the process of updating your Device Sync data model in an eligible app. 
+
+**Development Mode** is a Device Sync configuration setting that lets Sync
+automatically infer and update the server-side App Services schema based on your
+client-side changes. Instead of having to manually
+handle breaking changes to your data model, you can update your SDK object
+models and let Device
+Sync automatically handle the server-side updates.
+
+For more information on Development Mode and other Device Sync settings, refer 
+to :ref:`sync-settings`. 
+
+.. include:: /includes/note-dev-mode-production.rst
+
+For information on how to update data models once your app is in production
+(Development Mode is disabled), refer to :ref:`update-data-model-production-apps`.
+
+.. _update-data-model-development-mode-prerequisites:
+
+Prerequisites
+-------------
+
+To make breaking changes to your client-side object model using Development 
+Mode, your App Services App must meet the following requirements:
+
+- App is created *after* September 13, 2023, and has the following enabled: 
+  
+  - Device Sync. To learn how, refer to :ref:`enable-sync`.
+  - Development Mode. To learn how, refer to :ref:`enable-development-mode`.
+
+- App uses a minimum Device SDK version:
+
+  - C++ SDK v1.0.0
+  - Flutter SDK v1.6.0
+  - Java SDK v10.16.2
+  - Kotlin SDK v11.1.1
+  - .NET SDK v11.6.0
+  - Node.js SDK v12.2.0
+  - React Native SDK v12.2.0
+  - Swift SDK v10.42.2
+
+For apps created *before* September 13, 2023, you must manually update your
+object schema in the App Services UI. For more information, refer to
+:ref:`migrate-schema-partner-collection`.
+
+.. tip:: Contact Support to Enable in Older Apps
+   
+   You can contact support if you are interested in enabling this feature in an 
+   app created before September 13, 2023 that can meet the other requirements.
+
+Breaking Changes in Development Mode
+------------------------------------
+
+Non-breaking changes don't require any additional handling, even with
+Development Mode disabled. You can make these change to your server-side App
+Services schema or to your client-side object model.
+
+However, breaking changes (or destructive changes) require additional handling
+depending on where they are made. When Development Mode is enabled, and you make
+breaking changes to your client-side SDK object model, then Device Sync
+automatically handles the server-side updates for you.
```

Comment:
Consider highlighting (bold or italic) "server-side" and starting the next sentence with "However". Otherwise, a quick read says "...automatically handles the updates, but you must make your updates on the client..."